gramps_webapi/api/resources/base.py

```diff
@@ -58,6 +63,12 @@ def object_extend(self, obj: GrampsObject, args: Dict) -> GrampsObject:
             obj.extended = get_extended_attributes(self.db_handle, obj, args)
         return obj
 
+    def sort_objects(self, objs: List[str], args: Dict, locale) -> List:
+        """Sort the list of objects as needed."""
+        return sort_objects(
+            self.db_handle, self.gramps_class_name, objs, args, locale=locale
```

Comment:
I find it confusing that the argument in `sort_objects` is sometimes `objs`, sometimes `handles`. This is where type hints and keyword arguments are very useful to improve readability (and reduce the risk of bugs).

Reply:
Yeah I think I fixed some of the hints in a later commit not in this set.